scripts/install/aws_cloudshell_quickstart/07_cloudshell_setup_kubernetes.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# maybe read from inputs.tfvars?
+CLUSTER_NAME=cdc-nbs-sandbox
```

Comment:
This is an option, but there are 2 ways provided to supply the cluster name. Directly with a name variable or derived using the resource_prefix.

Reply:
definitely both for future improvements but wanted to capture current state